### ISSUE TITLE
TorXakis out of memory by big stautdef

### ISSUE THREAD

**Issue by petravandenbos** [TorXakis/TorXakis#702]
TorXakis runs out of memory, before having parsed the complete file.
[stautDefBig.txt](https://github.com/TorXakis/TorXakis/files/2005035/stautDefBig.txt)

**Comment by dnadales** [comment]
@pjljvandelaar @tretmans I have good news and bad news, the good news is that the new compiler can parse @petravandenbos 's model in about 3 seconds. The bad news is that I only have till July to finish the compiler :stuck_out_tongue_closed_eyes: 

The ugly news is that we might hit this problem in the compilation phase.

**Comment by pjljvandelaar** [comment]
Good news that the new parser is better....
For the rest: it is as it is.

**Comment by dnadales** [comment]
The new compiler takes 20 seconds to parse @petravandenbos file. So this can be fixed once the new compiler is merged into `develop`.

**Comment by pjljvandelaar** [comment]
🖒
